db.go

```diff
@@ -52,6 +52,14 @@ func init() {
 }
 
 func getOrCreateClient(countryCode, phoneNumber string) (*Client, error) {
+	if countryCode != getDigits(countryCode) {
+		return nil, errors.New("hey, countryCode should only contain digits")
+	}
+
+	if phoneNumber != getDigits(phoneNumber) {
```

Comment:
This will error when `phoneNumber` = "(647) 1234567", which is intended, right?

Reply:
OK, I just read https://github.com/hooligram/hooligram-server/issues/27
Question resolved.